doc/src/docs/InputOutputReference/01b-InputOutputReference.md

```diff
@@ -17398,7 +17386,7 @@ NOTE: branch object required only when tank use inlet nodes are used.
     Central HPWH Branch,     !- Name
     0,                       !- Maximum Flow Rate {m3/s}
     ,                        !- Pressure Drop Curve Name
-    WaterHeater:HeatPump,    !- Component 1 Object Type
```

Comment:
From this line down looks like a ZoneHVAC:EquipmentConnections object. The end of the HPWH object and beginning of the ZoneHVAC:EquipmentConnections object is missing here.